includes/Context.php

```diff
@@ -190,6 +190,12 @@ public function get_reference_site_url() {
 	public function get_reference_entity() {
 		// If currently in WP admin, run admin-specific checks.
 		if ( is_admin() ) {
+			// Support specific URL stats being checked in Site Kit dashboard details view.
+			$entity_url_query_param = 'googlesitekit-dashboard' === $this->input()->filter( INPUT_GET, 'page' ) ? $this->input()->filter( INPUT_GET, 'permaLink' ) : '';
```

Comment:
I'd suggest breaking this onto separate lines for readability. We can also make it a little more secure by requiring `permaLink` to be a valid URL.
```suggestion
			$entity_url_query_param = 'googlesitekit-dashboard' === $this->input()->filter( INPUT_GET, 'page' )
				? $this->input()->filter( INPUT_GET, 'permaLink', FILTER_VALIDATE_URL )
				: '';
```

Reply:
I opted for splitting into a separate if clause, I generally prefer these for readability over ternary unless the condition is really simple and line length remains short.